File: projects/5 projects/Desafio.java
import java.util.Scanner;

public class Desafio {
    public static void main(String[] args) {
        Scanner leitorEntrada = new Scanner(System.in);

        float valorSalario = leitorEntrada.nextFloat();
        float valorBeneficio = leitorEntrada.nextFloat();

        float valorImposto = 0f;

        if (valorSalario <= 1100f) {
            valorImposto = 0.05f * valorSalario;
        } else if (valorSalario <= 2500f) {
            valorImposto = 0.10f * valorSalario;
        } else {
            valorImposto = 0.15f * valorSalario;
        }

        float saida = valorSalario - valorImposto + valorBeneficio;

        System.out.println(String.format("%.2f", saida));

        leitorEntrada.close(); // boa prática
    }
}

File: projects/5 projects/desafio.kt
object ReceitaFederal {
    fun calcularImposto(salario: Double): Double {
        val aliquota = when {
            salario <= 1100.0 -> 0.05
            salario <= 2500.0 -> 0.10
            else -> 0.15
        }
        return aliquota * salario
    }
}

fun main() {
    val valorSalario = readLine()!!.toDouble()
    val valorBeneficio = readLine()!!.toDouble()

    val imposto = ReceitaFederal.calcularImposto(valorSalario)
    val saida = valorSalario - imposto + valorBeneficio

    println(String.format("%.2f", saida))
}
